app/src/main/java/space/taran/arknavigator/ui/fragments/GalleryFragment.kt

```diff
@@ -130,6 +130,10 @@ class GalleryFragment(
         }
     }
 
+    override fun setViewPagerInputEnabled(enabled: Boolean) {
```

Comment:
I understand, this name comes from `isUserInputEnabled`. It is clear what is "user input" in context of `ViewPager2` class, but on higher level it isn't. Let's rename it into something more straight-forward, like `enableScrolling` or whatever.